src/compose/utils.ts

```diff
@@ -388,11 +396,12 @@ export function addFeaturesFromLabels(
 			} as Dockerode.DeviceRequest),
 	};
 
-	_.each(features, (fn, label) => {
-		if (checkTruthy(service.config.labels[label])) {
-			fn();
+	for (const feature of Object.keys(features) as [keyof typeof features]) {
```

Comment:
```suggestion
	for (const feature: [keyof typeof features] of Object.keys(features)) {
```

Reply:
@CameronDiver `The left-hand side of a 'for...of' statement cannot use a type annotation.ts(2483)`

Reply:
Ah, that's a surprise. Ok no worries then